content/get-started/why-webpack.md

```diff
@@ -1,6 +1,31 @@
 ---
-title: Compared to Other Systems
+title: Why webpack?
+contributors:
+  - pksjce
+  - bebraw
+sort: 2
 ---
+
+You might be familiar with tools like Make, Grunt, Gulp, Browserify, or Brunch. Make is the oldest and the most known one as it's a general build tool and its history goes back to the 70s. More specialized *task runners*, like Grunt and Gulp, are focused on JavaScript and have healthy plugin ecosystems around them.
+
+## What Are Task Runners?
+
+Task runners literally make it easier to handle tasks, such as linting, building, or developing your project. Compared to *bundlers* like Browserify, Brunch, or webpack, they have a higher level focus. As you might guess from the term bundler, the goal of these tools is to achieve something lower level.
```

Comment:
In this paragraph I have some trouble understanding what ‘they,’ ‘these tools’ refer to.

Would rewording to something like this make the meaning more clear?

> Task runners literally make it easier to handle tasks, such as linting, building, or developing your project. Compared to *bundlers* like Browserify, Brunch, or webpack, task runners have a higher level focus. Bundlers, on the other hand, has a much more specific purpose.